Mathlib/Geometry/RingedSpace/LocallyRingedSpace.lean

```diff
@@ -74,37 +74,30 @@ structure Hom (X Y : LocallyRingedSpace.{u}) : Type _ where
   /-- the underlying morphism between ringed spaces -/
   val : X.toSheafedSpace ⟶ Y.toSheafedSpace
   /-- the underlying morphism induces a local ring homomorphism on stalks -/
-  prop : ∀ x, IsLocalRingHom (PresheafedSpace.stalkMap val x)
+  prop : ∀ x, IsLocalRingHom (val.stalkMap x)
 
 instance : Quiver LocallyRingedSpace :=
   ⟨Hom⟩
 
 @[ext] lemma Hom.ext' (X Y : LocallyRingedSpace.{u}) {f g : X ⟶ Y} (h : f.val = g.val) : f = g :=
   Hom.ext _ _ h
 
--- TODO perhaps we should make a bundled `LocalRing` and return one here?
--- TODO define `sheaf.stalk` so we can write `X.𝒪.stalk` here?
-/-- The stalk of a locally ringed space, just as a `CommRing`.
--/
-noncomputable def stalk (X : LocallyRingedSpace.{u}) (x : X) : CommRingCat :=
-  X.presheaf.stalk x
-
 -- Porting note (#10754): added this instance to help Lean realize stalks are local
 -- (so that `0 ≠ 1` works below)
-instance stalkLocal (x : X) : LocalRing <| X.stalk x := X.localRing x
+instance stalkLocal (x : X) : LocalRing <| X.presheaf.stalk x := X.localRing x
```

Comment:
This is already present on line 61 right? Not sure why we had it twice...

Reply:
Removed it.